js/menu.js

```diff
@@ -95,3 +97,73 @@ function toggleColorBox(){
 function toggleStatusBar(){
     document.getElementById('status-bar').classList.toggle('toggleStatusBar');
 }
+
+
+//when clicking the open option in the menu
+function openFile() {
+    //create a file input
+    const input = document.createElement("input");
+    input.type = "file";
+    input.click();
+    //detect image selected
```

Comment:
please make the file input exclusive to only images. When the file selection menu appears, it shows all the available files in the given directory. Although it doesn't break the application and since we're only drawing images, I guess it would be better to keep the file selection specific to images only.